### DISCUSSION TITLE
systemctl suspend & swaylock when lid is closed

### DISCUSSION THREAD

**Discussion by vyrx-dev** [prasanthrangan/hyprdots#1129] [Q&A] [answered]
# Trigger when the switch is turning off, Might cause bugs. Recommend to use logind instead.
#bindl= , switch:on:Lid Switch, exec, swaylock && systemctl suspend

I have read this in .conf file but how to use logind so without commenting the command laptop goes automatically in systemctl suspend and sway lock

**Comment by kRHYME7** [comment, marked as answer]
Try it first, the bug is device specific. It might be fine on your end.

>[!Note]
>It's convenient to just use the above method than using the systemd configuration 

Logind reference :
https://man.archlinux.org/man/logind.conf.5.en

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/b4aa2006-0da8-47c4-bbcb-1db0245236ca)

**Comment by fmolero69** [comment]
I want to use this same post for a similar issue!
Where can I configure so that when I stop using my PC for 20 minutes, the monitor goes into standby mode, or the PC goes to sleep?

**Reply by kRHYME7** [reply]
https://github.com/hyprwm/hypridle this 


or sue swayidle

**Reply by rubiin** [reply]
I want this too. Sometimes my cat plays on my keyboard as the laptop doesnt go to idle mode when inactive for a certain time.
